vaadin-combo-box-item.html

```diff
@@ -16,6 +16,14 @@
       :host([hidden]) {
          display: none;
       }
+
+      :host([disabled]:hover) {
+        background: none !important;
```

Comment:
!important should be avoided. Use `:not([disabled])` instead for the original hver effect instead.